### PR TITLE
Feature/editorial review updates

### PR DESCRIPTION
**Pull request by stuartjennings-up**
https://trello.com/c/G62mUvCT/190-3-updates-to-editorial-review-process

### CODE REVIEW

File: src/editorialreview/tests.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+from django.test import TestCase
+from django.test.client import Client
+
+from editorialreview import models
+from core import models as core_models
+from review import models as review_models
+from submission import models as submission_models
+
+
+class UserFactory(factory.Factory):
```
Comment (fradeve) [inline src/editorialreview/tests.py:14]: I like the idea of having all factories living in a dedicated factory file in the test module, so you can import them like this:

```
from editorialreview.test.factories import *
```

But this is allright for now

File: src/editorialreview/views.py [hunk 2]
```diff
@@ -34,7 +34,8 @@
 
 @is_editor
 def add_editorial_review(request, submission_type, submission_id):
-    """ Create a new editorial review. """
+    """Create a new editorial review."""
```
Comment (fradeve) [inline src/editorialreview/views.py:37]: :+1:

File: src/editorialreview/views.py [hunk 3]
```diff
@@ -438,7 +443,18 @@ def editorial_review(request, review_id):
                 default='The publishers',
             )
 
-            for editor in submission.book_editors.all():
+            try:
+                series_editor_email = [submission.series.editor.email]
+            except:
```
Comment (fradeve) [inline src/editorialreview/views.py]: Can we have a more detailed exception?

**Review by fradeve — APPROVED** [review]
Once the exception is specified, this LGTM

File: src/editorialreview/tests.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            'editorialreviewer'
+        )
+        self.assertEqual(
+            self.editorialreviewer.email,
+            'editorialreviewer@email.com'
+        )
+
+    def test_ed_reviewer_download_proposal(self):
+        self.editorialreview = ProposalEditorialReviewFactory.create()
+        print(vars(self.editorialreview.content_type))
```
Comment (terrasea2) [inline src/editorialreview/tests.py]: Why is there a print statement here? Tests shouldn't really have prints in them.

File: src/editorialreview/views.py (hunk 3 above)
Comment (terrasea2) [reply src/editorialreview/views.py]: I agree with @fradeve . Just having except is not that useful, if something goes wrong, we should at least log it

File: src/templates/core/proposals/proposal_editorial_review.html [hunk 5]
```diff
@@ -10,23 +10,50 @@
...
+              {% if review.user.middle_name %}
+                {{ review.user.middle_name }}
+              {% endif %}
+              {{ review.user.last_name }}
+          </a>
+          <sup>
+              <a href="core/email.html"
+                 onclick="return popitup('/email/proposal/{{ proposal.id }}/user/{{ review.user.id }}/',
+                 'proposal_{{ review.user.id }}')"
+                 class="fa fa-envelope-o btn btn-xs btn-default">
```
Comment (terrasea2) [inline src/templates/core/proposals/proposal_editorial_review.html:33]: I assume this is supposed to be an empty anchor tag?

File: src/templates/core/proposals/proposal_review_block.html [hunk 6]
```diff
@@ -2,32 +2,103 @@
...
+        {{ reviewer.user.first_name }}
+        {% if author.middle_name %}
+          {{ reviewer.user.middle_name}}
+        {% endif %}
+          {{ reviewer.user.last_name }}
+      </a>
+      <sup>
+        <a href="core/email.html"
+           onclick="return popitup('/email/proposal/{{proposal.id}}/user/{{reviewer.user.id}}/','proposal_{{reviewer.user.id}}')"
+           class="fa fa-envelope-o btn btn-xs btn-default">
```
Comment (terrasea2) [inline src/templates/core/proposals/proposal_review_block.html:27]: Again, empty tag, which I assume there is a reason for?

**Review by terrasea2 — CHANGES_REQUESTED** [review]
Mostly small things, and even then mostly questions about things which look odd

File: src/editorialreview/tests.py (hunk 4 above)
Comment (stuartjennings-up) [reply src/editorialreview/tests.py]: Wasn't meant to be in there- good spot

File: src/templates/core/proposals/proposal_editorial_review.html (hunk 5 above)
Comment (stuartjennings-up) [reply src/templates/core/proposals/proposal_editorial_review.html:33]: It's an email icon that links to a separate email page

**Pull request merged**